src/components/ToDoItemComp.vue

```diff
@@ -1,32 +1,223 @@
 <template>
-  <div class="grid grid-rows-2 gap-2 text-center border-2 border-black">
-    <div class="grid grid-cols-2">
-      <p>{{ props.item.title }}</p>
-      <p>{{ props.item.priority }}</p>
+  <div class="text-center border-2 border-black mx-3.5 rounded-2xl">
+    <div
+      v-if="showPopUp"
+      class="fixed inset-0 items-center justify-center bg-black/75 flex"
+    >
+      <div
+        class="bg-white z-20 border-black border-2 rounded-xl flex flex-col items-center lg:w-2/5 w-4/5"
+      >
+        <div class="text-lg text-white font-semibold rounded-full bg-red mt-2">
+          <p class="mx-5">Warning</p>
+        </div>
+        <p class="m-2">
+          Are you sure you want to delete this item? This operation is permanent
+          and you will not be able to undo this action!
+        </p>
+        <div class="flex flex-row space-x-6 mb-2 font-semibold">
+          <button
+            class="rounded-full bg-green px-2 text-white"
+            @click="removeTodo"
+          >
+            Yes
+          </button>
+          <button
+            class="rounded-full bg-red px-2 text-white"
+            @click="showPopUp = false"
+          >
+            No
+          </button>
+        </div>
+      </div>
     </div>
-    <div>
-      <p>{{ props.item.text }}</p>
-    </div>
-    <div>
-      <button>Save</button>
-      <button class="border-2" @click="removeTodo(props.index)">Remove</button>
+    <div class="flex flex-col m-[18px]">
+      <div class="flex flex-row justify-between items-center">
+        <input
+          v-model="currentTitle"
+          type="text"
+          placeholder="Title"
+          class="w-[197px] font-semibold text-lg lg:text-4xl"
+          :class="showDropDown ? 'opacity-30' : ''"
+          :disabled="showDropDown"
+        />
+        <div class="flex flex-row space-x-[5px] items-center lg:hidden">
+          <div
+            class="p-1 bg-blue rounded-full h-fit"
+            :class="
+              currentPriority === Priority.low ? 'border border-black' : ''
+            "
+            @click="changePriority(Priority.low)"
+          ></div>
+          <div
+            class="p-1 bg-yellow rounded-full h-fit"
+            :class="
+              currentPriority === Priority.medium ? 'border border-black' : ''
```

Comment:
Create a function for this, which will have a parameter named priority and it will return something like currentPriority === Priority[priority] ? 'border border-black' : ''